### PR TITLE
Add time to export success file name

### LINKED ISSUE #1714: [Export] Summary filename to include time

### PR DESCRIPTION
**Pull request by itaiad200**
closes #1714

### CODE REVIEW

**Review by johnnyaug — APPROVED** [review]

File: clients/spark/core/src/main/scala/io/treeverse/clients/Exporter.scala [hunk 1]
```diff
@@ -102,7 +104,8 @@ class Exporter(spark : SparkSession, apiClient: ApiClient, filter: KeyFilter, re
 
   private def writeSummaryFile(success: Boolean, commitID: String, content : String) = {
     val suffix = if(success) "SUCCESS" else "FAILURE"
-    val dstPath = resolveURL(new URI(dstRoot), s"EXPORT_${commitID}_${suffix}")
+    val time = ZonedDateTime.now(ZoneOffset.UTC).format(DateTimeFormatter.ISO_INSTANT)
```
Comment (arielshaqed) [inline clients/spark/core/src/main/scala/io/treeverse/clients/Exporter.scala:107]: I don't understand the conversions into a fake timezone.  The requirement has no timezone, so why does the code include one?  [`ISO_INSTANT`](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_INSTANT) _is_ "in UTC", per documentation.

```scala
scala> val t = time.Instant.now()
val t: java.time.Instant = 2021-04-07T08:31:10.794768Z

scala> ISO_INSTANT.format(t)
val res7: String = 2021-04-07T08:31:10.794768Z

```

```suggestion
    val t = ISO_INSTANT.format(time.Instant.now()
```

**Review by arielshaqed — CHANGES_REQUESTED** [review]

File: clients/spark/core/src/main/scala/io/treeverse/clients/Exporter.scala (hunk 1 above)
Comment (itaiad200) [reply clients/spark/core/src/main/scala/io/treeverse/clients/Exporter.scala]: Done.

File: clients/spark/core/src/main/scala/io/treeverse/clients/Exporter.scala [hunk 2]
```diff
@@ -104,7 +104,7 @@ class Exporter(spark : SparkSession, apiClient: ApiClient, filter: KeyFilter, re
 
   private def writeSummaryFile(success: Boolean, commitID: String, content : String) = {
     val suffix = if(success) "SUCCESS" else "FAILURE"
-    val time = ZonedDateTime.now(ZoneOffset.UTC).format(DateTimeFormatter.ISO_INSTANT)
+    val time = DateTimeFormatter.ISO_INSTANT.format(ZonedDateTime.now())
```
Comment (arielshaqed) [inline clients/spark/core/src/main/scala/io/treeverse/clients/Exporter.scala:107]: StillLGTM, but still wary of this code being "zoned" (to coin a word).  Just get the current instant, even with [`java.time.Clock.systemUTC`](https://docs.oracle.com/javase/8/docs/api/java/time/Clock.html#systemUTC--)`.instant()`.

Timezones have no place in code, *except* when formatting a local time for human consumption or arithmetic with "civil" durations (years, months, and often days).

**Review by arielshaqed — APPROVED** [review]
Still good, still need to get rid of all mention of timezones (in code that does not need them).

**Pull request merged**